posgram/Repository/IRepository/IUserRepository.cs

```diff
@@ -10,6 +10,6 @@ public interface IUserRepository
         Task Create(User entity);
         Task Update(User entity);
         Task Delete(User entity);
-        bool IsUserExsits(string email, string username);
+        bool IsUserExists(string email, string username);
```

Comment:
Duplication of `user`, because we have already named this repository as `IUserRepository`. Maybe it will be better to rename to just:
```suggestion
        bool IsExists(string email, string username);
```

Reply:
Ok